viaMover/web-app
# [Feature] Olympus avatar

What was done
* Added new styles for Olympus NFT
* Added a new localized string for Olympus NFT
* Added new persistence behavior
* Added new types for avatars
* Added helpers to ease differentiation between avatars
* Updated `header-balance-avatar` component
* Updated `loadAvatar` action
* Chained `loadAvatar` action with `loadNftInfo` because they have a dependency chain involved during the wallet initialization period
* Performed a small cleanup on the styles & logic
* Added `nft/isLoading` to the `isWalletReady` getter
* Updated RootStoreState to be an interface to eliminate Vuex warnings and achieve better code completion

How to test
* Before checkout to this branch prepare another account, launch an application just to have a separate `localStorage` entry with the old signature (important!)
  * Change an account to the desired testing one
  * Checkout to this branch
  * Load a wallet
* Modify`nft/hasOlympus` getter to always return true
  * Check that avatar is changed to the Olympus avatar (simulate that the other user received their NFT during mint/receive process)
  * Refresh the page
  * Should be restored with Olympus avatar (the last used)
* Toggle the avatar by clicking on it
  * Should work just fine
  * Should cycle over `20` default user avatars as well as newly added one
* Toggle the avatar to the non-Olympus one
  * Refresh the page
  * Should be restored as non-Olympus (the last used)
* Modify`nft/hasOlympus` getter to always return false (simulate that the user send their NFT to someone)
  * Check that the custom avatar was replaced with the random one
  * Cycle over user avatars, there should be no Olympus avatar
